tests/unit_test/config.py

```diff
@@ -0,0 +1,292 @@
+        and data_item["name"] == "project_spend_cost"
+    ):
+        cloudwatch_project_spend_cost_description = data_item["values"]["description"]
+        break
+
+cloudwatch_project_spend_cost_cron = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "project_spend_cost"
+    ):
+        cloudwatch_project_spend_cost_cron = data_item["values"]["schedule_expression"]
+        break
+
+cloudwatch_resource_list = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "resource_list"
+    ):
+        cloudwatch_resource_list = data_item["values"]["name"]
+        break
+
+cloudwatch_resource_list_description = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "resource_list"
+    ):
+        cloudwatch_resource_list_description = data_item["values"]["description"]
+        break
+
+cloudwatch_resource_list_cron = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "resource_list"
+    ):
+        cloudwatch_resource_list_cron = data_item["values"]["schedule_expression"]
+        break
+
+ec2_ami = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_instance" and data_item["name"] == "this":
+        ec2_ami = data_item["values"]["ami"]
+        break
+
+ec2_az = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_subnet" and data_item["name"] == "public_subnet":
+        ec2_ami = data_item["values"]["availability_zone"]
+        break
+
+iam_role_project_spend_cost_name = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_iam_role" and data_item["name"] == "ProjectSpendCost":
+        iam_role_project_spend_cost_name = data_item["values"]["name"]
+        break
+
+iam_role_project_spend_cost_assumed_policy = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_iam_role" and data_item["name"] == "ProjectSpendCost":
+        iam_role_project_spend_cost_assumed_policy = data_item["values"][
+            "assume_role_policy"
+        ]
+        break
+
+iam_role_resource_list_name = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_iam_role"
+        and data_item["name"] == "resource_list_service_role"
+    ):
+        iam_role_resource_list_name = data_item["values"]["name"]
+        break
+
+iam_role_resource_list_assumed_policy = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_iam_role"
+        and data_item["name"] == "resource_list_service_role"
+    ):
+        iam_role_resource_list_assumed_policy = data_item["values"][
+            "assume_role_policy"
+        ]
+        break
+print(iam_role_resource_list_assumed_policy)
```

Comment:
can we use logging instead of print

Reply:
Resolved in [d67f4bc](https://github.com/XgridInc/xc3/pull/18/commits/d67f4bcc757b95ad043c435e8c4acb4dd33c6a76)